cookienotice/cookienotice.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+/**
+ * Name: Cookie Notice
+ * Description: Configure, show and handle a simple cookie notice
+ * Version: 1.0
+ * Author: Peter Liebetrau <https://socivitas/profile/peerteer>
+ * 
+ */
+use Friendica\Core\Addon;
```

Comment:
This `use` statement doesn't seem to be used anymore.

Reply:
done